code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -155,16 +160,10 @@
 			destroy_objective.owner = owner
 			destroy_objective.find_target()
 			add_objective(destroy_objective)
-		else if(prob(30))
-			var/datum/objective/maroon/maroon_objective = new
-			maroon_objective.owner = owner
-			maroon_objective.find_target()
-			add_objective(maroon_objective)
+		else if (prob(50))
```

Comment:
Should definitely be like 30 or so. Protect should not be a primary objective. Maroon/kill is much better.
```suggestion
		else if (prob(30))
```